common/network-yarn/src/main/java/org/apache/spark/network/yarn/YarnShuffleService.java

```diff
@@ -321,6 +326,7 @@ public ByteBuffer getMetaData() {
    * overrode and called when Hadoop version is 2.5+ and NM recovery is enabled, otherwise we
```

Comment:
Could you update this comment since it's out of date now?

Reply:
Sure I will.